src/main/scala/gestalt/Quote.scala

```diff
@@ -51,8 +52,10 @@ abstract class Quote(val t: Toolbox, val toolboxName: String) {
     t.Select(qual, parts.last)
   }
 
-  // lifts: m.Tree => t.Tree[t.Tree]
-
+  /** {{{
+    * AnyTree = t.Tree | t.TypeTree
+    * (trees: Seq[m.Tree[A]]) => (t.Tree[Seq[String]] | t.Tree[Seq[AnyTree[?]]])
+    * }}}*/
```

Comment:
Thanks for the explanation, it makes sense. We may have potential bugs related to `interpolate`, making the types correct is tricky. Anyway, the comments are just minor issues, feel free to change the form you prefer and keep them simple.